scripts/setup-bridging-contracts.js

```diff
@@ -126,14 +128,69 @@ async function setupBridging(homeRpcUrl, foreignRpcUrl) {
   // Start the bridge service
   console.log(`Home RPC Url: ${homeRpcUrl}`);
   console.log(`Foreign RPC Url: ${foreignRpcUrl}`);
-  const bridgeMonitor = new MockBridgeMonitor(
+  const guardianSpy = new MockGuardianSpy(
     homeRpcUrl,
     foreignRpcUrl,
     homeBridge.address,
     foreignBridge.address,
     homeColonyBridge.address,
     foreignColonyBridge.address,
   ); // eslint-disable-line no-unused-vars
+
+  // TODO: Start the bridge monitor
+  console.log("Starting bridge monitor");
+
+  // Write config file
+  const config = `
```

Comment:
Can we not import a config from somewhere?

Reply:
I think that's tricky because of our desire to use the `[wormhole.CHAIN_ID_ARBITRUM_SEPOLIA]` syntax. We can't import and `JSON` parse it (because it's not valid JSON), and if we imported it as a string, we'd have to meticulously replace the keys based on line / character number.

If you have a suggestion here, I'm all ears!

Reply:
I guess regex?

Reply:
Ok, I'm not too broken up about it if there's good reasons for doing it this way.